### PR TITLE
feat (sidatuk): slicing media section

### PR DESCRIPTION
**Pull request by naufalihsank**
### Related
- [S47A2 - Layanan SIDATUK](https://app.clickup.com/t/2na0hg1)
### Preview
**Mobile**
image
**Tablet**
image
**Desktop**
image
### Evidence
title: slicing media section
project: Portal Jabar
participants: @naufalihsank @Ibwedagama @agunghide @doohanas

### CODE REVIEW

File: components/Layanan/Item/Media/index.vue [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<template>
+  <section class="flex flex-col sm:flex-row xl:flex-col flex-wrap gap-4 xl:gap-6 mt-4 md:mt-6 lg:mt-12 xl:mt-6 sm:h-[964px] lg:h-[1004px] xl:h-[557px]">
+    <!-- Youtube Video -->
+    <section class="bg-gray-900 w-full xl:w-[816px] h-[156px] sm:h-[383px] rounded-xl flex items-center justify-center">
+      <lite-youtube
+        :videoid="videoId"
+        class="w-full h-full bg-gray-900 rounded-xl"
+      />
```
Comment (Ibwedagama) [inline components/Layanan/Item/Media/index.vue:8]: Hello @naufalihsank , is this library safe to use? I'm concern it is not being maintained by the author for 2 years now 🤔 

Should we use Youtube's own `iframe` embed tag?

```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/BwuLxPH8IDs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment (Ibwedagama) [reply components/Layanan/Item/Media/index.vue:8]: But I see this library claimed to be more performant than using `iframe` and that's can help us achieve better performance for this page 🤔 

https://user-images.githubusercontent.com/39191/68095561-5d2ba380-fe5f-11e9-9393-e2206a64c8bf.png

**Review by doohanas — APPROVED** [review]

File: components/Layanan/Item/Media/index.vue (hunk 1 above)
Comment (naufalihsank) [reply components/Layanan/Item/Media/index.vue:8]: i tried to search another package cause i failed to use `iframe` previously 🤣 (but it runs normally now wkwk)

then i read [this](https://betterprogramming.pub/4-steps-to-embed-youtube-in-nuxt-js-with-a-markdown-file-9d5fe9a0167a) article that recommend to use that package

but beside that package is not being maintained for 2 years, another const is that this `lite-youtube` component cannot receive raw youtube link into the `src` props

soo how ? 🤔

**Review by agunghide — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

File: components/Layanan/Item/Media/index.vue (hunk 1 above)
Comment (naufalihsank) [reply components/Layanan/Item/Media/index.vue:8]: yes, i already implement that splitting method to collect the video's `id`

sure and we should monitor this library's performance for the time being

**Pull request merged**